test/js/service/github.service.spec.js

```diff
@@ -0,0 +1,32 @@
+describe("github.service", function() {
+
+  var mockWindow, mockNotification, sampleGithubService;
+  beforeEach(function(){
+    module(function($provide){
+      $provide.service('$window', function(){ 
+        this.location.href = jasmine.createSpy('location.href');
+      });
+      $provide.service('notifications', function(){
+        this.apiService.get = jasmine.createSpy('apiService.get');
+      });
+    });
+    module('services');
+  });
+
+  beforeEach(inject(function($window, notifications, githubService){
+    mockWindow=$window;
+    mockNotification=notifications;
+    sampleGithubService=githubService;
+  }));
+
+  it('should show notifications', function(){
+    var repo="Some message";
+    sampleGithubService.notifications(repo);
```

Comment:
this githubService.notifications will call a $http.get and return a promise. 
so you will need to handle it as such
you can use the $httpBackend to mock the response, like this:
```
$httpBackend.expect('GET', 'github/notifications?repo=repo')
    .respond({
      "success": stubResponse
    });
```

and you can assert  on the promise response
```
promise.then(function(response) {
   expect(response).toEqual(stubResponse);
})
```